app/_how-tos/collect-dev-portal-audit-logs-with-sumologic.md

```diff
@@ -0,0 +1,105 @@
+
+prereqs:
+  inline:
+    - title: Dev Portal
+      include_content: prereqs/dev-portal-app-reg
+      icon_url: /assets/icons/dev-portal.svg
+    - title: Dev Portal ID
+      content: |
+        From the overview of the [Dev Portal you created](#dev-portal), get your Dev Portal ID and export it to your environment:
+
+        ```sh
+        export PORTAL_ID='YOUR DEV PORTAL ID'
+        ```
+      icon_url: /assets/icons/dev-portal.svg
+    - title: SumoLogic SIEM provider
+      include_content: /prereqs/sumologic-siem
+
+tools:
+  - konnect-api
+
+cleanup:
+  inline:
+    - title: Clean up {{site.konnect_short_name}} environment
+      include_content: cleanup/platform/konnect
+      icon_url: /assets/icons/gateway.svg
+---
+
+## Set up the audit log destination
+
+Create an audit log destination by sending a `POST` request to the [`/audit-log-destinations`](/api/konnect/audit-logs/v2/#/operations/create-audit-log-destination) endpoint with the connection details for your SIEM vendor:
+
+```sh
+curl -i -X POST https://global.api.konghq.com/v2/audit-log-destinations \
+--header "Content-Type: application/json" \
+--header "Authorization: Bearer $KONNECT_TOKEN" \
+--json '{
+    "endpoint": "'$SIEM_ENDPOINT'",
+    "authorization": "'$SIEM_TOKEN'",
+    "log_format": "cef",
+    "name": "Example destination"
+}'
+```
+
+Export the ID of the new destination to your environment:
+
+```sh
+export DESTINATION_ID='YOUR DESTINATION ID'
+```
+
+## Enable the webhook on your Dev Portal
+
+Create a webhook by sending a `PATCH` request to the [`/portals/{portalId}/audit-log-webhook`](/api/konnect/portal-management/v3/#/operations/update-portal-audit-log-webhook) endpoint with the audit log destination:
+
+```sh
+curl -i -X PATCH https://us.api.konghq.com/v3/portals/$PORTAL_ID/audit-log-webhook \
+ --header "Content-Type: application/json" \
+ --header "Authorization: Bearer $KONNECT_TOKEN" \
+ --json '{
+     "audit_log_destination_id": "'$DESTINATION_ID'",
+     "enabled": true
+ }'
+```
+
+Webhooks are triggered via an HTTPS request using the following retry rules:
+
+- Minimum retry wait time: 1 second
+- Maximum retry wait time: 30 seconds
+- Maximum number of retries: 4
+
+A retry is performed on a connection error, server error (`500` HTTP status code), or too many requests (`429` HTTP status code).
+
+## Validate
+
+To validate that the webhook is configured correctly, you can log in to your Dev Portal with the account you created in the [prerequisites](#dev-portal). This should trigger a log in SumoLogic. Sometimes it can take a minute to populate the logs.
+
+In the SumoLogic UI, navigate to the [log search](https://service.sumologic.com/log-search) and search for `_sourcecategory="Http Input"`. You should see logs like the following:
```

Comment:
```suggestion
In the SumoLogic UI, navigate to the [log search](https://service.sumologic.com/log-search) and search for `_source=Konnect`. You should see logs like the following:
```
The search example didn't work for me. The one I added did, but I think that was because I named my source `Konnect` in SumoLogic.